test/check-machines-snapshots

```diff
@@ -238,6 +239,11 @@ class TestMachinesSnapshots(machineslib.VirtualMachinesCase):
                     print("------ end snapshot XML -------")
                     raise
 
+            def revert(self):
+                b.click(f"#vm-subVmTest1-snapshot-{self.snap_num}-revert")
+                b.click('.pf-v5-c-modal-box__footer button:contains("Revert")')
+                b.wait_not_present('.pf-v5-c-modal-box__footer button:contains("Revert")')
```

Comment:
This is weird. Shouldn't this ensure that the whole dialog goes away? `b.wait_not_present(".pf-v5-c-modal-box")`

Reply:
Yeah, I think I cargo-culted it. I have a look.

Reply:
Done.